meinberlin/apps/contrib/templates/meinberlin_contrib/includes/form_field.html

```diff
@@ -2,20 +2,20 @@
 
 <div class="form-group {% if add_class %}{{ add_class }}{% endif %}">
     <label for="{{ field.id_for_label }}" {% if tabindex %} tabindex="{{ tabindex }}"{% endif %}>
-        {{ field.label }}{% if field.field.required %}<span role="presentation" title="{% translate 'This field is required' %}">*</span>{% endif %}
+        {{ field.label }}{% if field.field.required %}<span role="presentation" title="{% translate "This field is required" %}">*</span>{% endif %}
         {% block after_label %}
-        {% endblock %}
+        {% endblock after_label %}
     </label>
-    {% if field.help_text %}
+    {% if field.help_text and not "password" in field.help_text %}
```

Comment:
maybe `and not field.name == "password1"` is a bit safer? just in case any other help_texts might contain the word password?

Reply:
ah, sorry, not `password1` but `password`

Reply:
It doesn't work with exact matching...

Reply:
oh the field.name.. ok right